hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -983,6 +994,15 @@ void freeSpace(final String why) {
       // Scan entire map putting bucket entry into appropriate bucket entry
       // group
       for (Map.Entry<BlockCacheKey, BucketEntry> bucketEntryWithKey : backingMap.entrySet()) {
+
+        if (coldFiles != null
+          && coldFiles.containsKey(bucketEntryWithKey.getKey().getHfileName())) {
+          int freedBlockSize = bucketEntryWithKey.getValue().getLength();
+          evictBlockIfNoRpcReferenced(bucketEntryWithKey.getKey());
```

Comment:
`evictBlockIfNoRpcReferenced` returns a boolean indicating whether the eviction was successful or not. We should check that boolean in orther to decide if we should increment `bytesFreed`.

Reply:
ack!